### ISSUE TITLE
issue with security question on tablet and phone

### ISSUE THREAD

**Issue by jackmaessen** [taufik-nurrohman/flat-file-guestbook#1]
There seems to be an issue with the security question when using tablet or phone to submit a comment. It always echo's: "Wrong math answer"

**Comment by taufik-nurrohman** [comment]
So, it is okay with the desktop? Have you disabled the cookie on your tablet and phone?

**Comment by jackmaessen** [comment]
No i did not disable cookie on tablet or phone. On desktop, everything works fine! I will try to disable cookies on phone and tablet.

**Comment by taufik-nurrohman** [comment]
No, I mean, disabling the cookie will disable the PHP session ID which will break the captcha answer.
